install/receivers.csv

```diff
@@ -151,6 +151,7 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40033,KAPT,2020-02-11T20:15:38Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40034,TGTK,2020-01-29T21:48:38Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40069,SCTY,2020-06-23T01:06:32Z,2020-10-27T23:29:07Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,MKNO,2020-02-12T03:08:38Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRIMBLE ALLOY,5951R40056,RGHL,2021-06-01T21:46:08Z,9999-01-01T00:00:00Z
```

Comment:
can you please update the sn to `5946R40026`?

Reply:
Done